javascript/javascript3/week2/Exercises.js

```diff
@@ -0,0 +1,113 @@
+async function getMovies() {
+  try {
+    const response = await fetch(
+      "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+    );
+    const movies = await response.json();
+    console.log(movies);
+    const badMovies = movies.filter((movie) => movie.rating < 6);
+    console.log(badMovies);
+    const badMoviesSince2000 = badMovies.filter((movie) => movie.year >= 2000);
+    console.log(badMoviesSince2000);
+  } catch (error) {
+    console.log(error);
+  }
+}
+
+getMovies();
+
+// Promise that resolves after set time
+
+function resolvedAfterSetTime(resolveAfter) {
+  return new Promise(function (resolve) {
+    setTimeout(function () {
+      resolve("I am called asynchronously");
+    }, resolveAfter * 1000);
+  })
+    .then(function (result) {
+      console.log(result);
+    })
+    .catch(console.log.bind(console));
```

Comment:
Tiny detail but I think the exercise meant for these to be added outside of the function. The function only works to delay the execution.

Also, the call to `bind` isn't strictly needed :)